src/components/AppLayout/Header/components/Layout.tsx

```diff
@@ -19,24 +19,24 @@ import { useSelector } from 'react-redux'
 import { OVERVIEW_EVENTS } from 'src/utils/events/overview'
 import Track from 'src/components/Track'
 import Notifications from 'src/components/AppLayout/Header/components/Notifications'
-import AnimatedLogo from 'src/components/AppLayout/Header/components/AnimatedLogo'
+import styled from 'styled-components'
 import SafeTokenWidget, { getSafeTokenAddress } from './SafeTokenWidget'
 import { _getChainId } from 'src/config'
 
 const styles = () => ({
   root: {
-    backgroundColor: 'white',
+    backgroundColor: "black",
     borderRadius: sm,
-    boxShadow: 'rgb(40 54 61 / 18%) 1px 2px 10px 0px',
+    border: "2px solid #06fc99",
     marginTop: '11px',
     minWidth: '280px',
     padding: 0,
   },
   summary: {
     alignItems: 'center',
-    backgroundColor: 'white',
+    backgroundColor: 'black',
     flexWrap: 'nowrap',
-    height: headerHeight,
+    height: '5rem',
```

Comment:
Make sure this resizes on all display sizes; preferably dont hard code height use headerHeight